app/views/profile/projects.html.haml

```diff
@@ -17,3 +17,24 @@
     .row
       .col-md-12
         =render current_user.participating_projects
+
+  -if current_user.projects.empty? and current_user.participating_projects.empty?
+    .row
+      .col-md-2
+      .col-md-8
+        .panel.panel-default
+          .panel-heading
+            %h3= t('profile.projects')
+
+          .panel-body
+            %h4= t('profile.no_projects')
+
+            %p
+              #{t('profile.all_projects_pre')}
```

Comment:
In HAML views, you can simply write `=t('...')`. No need for the `#{...}` wrapping.